### PR TITLE
Job Improvement Project - Kagerou & Oboro

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: #4424

* **Server Mode**: Renewal

* **Description of Pull Request**: 
  * kRO Changelog: http://ro.gnjoy.com/news/notice/View.asp?BBSMode=10001&seq=7081

### CODE REVIEW

**Comment by Badarosk0** [comment]
Shadow Warrior is not currently applying it's 20% Bonus Damage to the following skills : Kunai Explosion, Kunai Splash, Cross Slash and Swirling Petal.

**Comment by OptimusM** [comment]
@Badarosk0 
U can try change the current modifier

```
			if (sc && sc->data[SC_KAGEMUSYA])
				skillratio += 20;
```

by this one

```
			if (sc && sc->data[SC_KAGEMUSYA])
				skillratio += skillratio * 20 / 100;
```

**Comment by RagnaWay** [comment]
`t_tickime` undeclared identifier

**Comment by admkakaroto** [comment]
In level 10, Kage Charms incremet some skills in 1000%

image png eeff985fc1b1e7350f122d7d7a27980d

**Comment by teededung** [comment]
Missing update for Wind Charm:
- - Increases Lightning Jolt (NJ_RAIGEKISAI) damage from 15% to 20% per 1 charm.
- - Increases First Wind (NJ_KAMAITACHI) damage from 10% to 100% per 1 charm.

File: db/re/skill_db.yml [hunk 1]
```diff
@@ -29833,26 +29832,54 @@ Body:
...
+        Size: 5
       - Level: 5
+        Size: 5
+      - Level: 6
+        Size: 6
+      - Level: 7
+        Size: 6
+      - Level: 8
+        Size: 6
+      - Level: 9
```
Comment (Angelic234) [inline db/re/skill_db.yml]: MIss Level 10 info.

**Comment by admkakaroto** [comment]
Not compiling:
`\src\map\skill.cpp(7552,37): error C2065: 'sc': undeclared identifier`

Problem in this line:

> if (skill_id == SJ_STAREMPEROR && sc && sc->data[SC_DIMENSION]) {

**Comment by aleos89** [comment]
Is there a reason Star Emperor skills will interfere with this PR? Github isn't showing any merge conflicts.

**Comment by admkakaroto** [comment]
well, it shouldn't, i used git's "apply patch" function and tried to recompile and received this msg, i will try to apply manually this PR and try to compile.

**Comment by admkakaroto** [comment]
@aleos89 I applied it manually but received the same error.

**Pull request merged**